doc/source/conf.py

```diff
@@ -134,11 +135,11 @@ def check_sphinx_version(expected_version):
 #
 # The short X.Y version.
 # version = '1.0.6'
-from drizzlepac import __version__, __version_date__
+from drizzlepac import __version__
 version = __version__
 # The full version, including alpha/beta/rc tags.
 # release = '1.0.6 (14-Aug-2012)'
-release = "{:s} ({:s})".format(__version__, __version_date__)
+release = "{:s} ".format(__version__)
```

Comment:
I would suggest:
```python
from drizzlepac import __version__ as version
release = version
```
According to item `#5` in https://www.python.org/dev/peps/pep-0396/#specification `__version__` should be a string.